src/StatusQ/Popups/StatusMenuItemDelegate.qml

```diff
@@ -158,7 +157,9 @@ MenuItem {
                   return Theme.palette.directColor1
             }
         }
-        font.pixelSize: 13
+        font.pixelSize: !!statusPopupMenuItem.action.fontSettings ? statusPopupMenuItem.action.fontSettings.pixelSize : 13
+        font.bold: !!statusPopupMenuItem.action.fontSettings ? statusPopupMenuItem.action.fontSettings.bold : false
+        font.italic: !!statusPopupMenuItem.action.fontSettings ? statusPopupMenuItem.action.fontSettings.italic : false
```

Comment:
Are these checks really necessary? `StatusMenuItem.action` should have defined default `fontSettings`, shouldn't it?

Reply:
Actually, they are needed. Without it, there are some error messages that it is not existing. I suspect that this is because of some usages of Action instead of StatusMenuItem in StatusPopupMenu (I saw a few places).